Patch by Robert Scheck <robert@fedoraproject.org> for sendmail >= 8.15.1 which ensures that the
loopback IPv6 address is handled the same way like the IPv4 one. However since sendmail >= 8.15.1
uncompressed IPv6 addresses are used by default thus the previously used IPv6:::1 gets changed to
now IPv6:0:0:0:0:0:0:0:1 (if not compiled with -DIPV6_FULL=0 for the prior format). For further
information see also:

- https://bugzilla.redhat.com/show_bug.cgi?id=549217
- https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=618570
 
--- sendmail-8.15.1/cf/feature/block_bad_helo.m4		2014-03-24 15:29:09.000000000 +0100
+++ sendmail-8.15.1/cf/feature/block_bad_helo.m4.ipv6-bad-helo	2015-03-20 01:22:42.000000000 +0100
@@ -16,3 +16,5 @@
 define(`_BLOCK_BAD_HELO_', `')dnl
 RELAY_DOMAIN(`127.0.0.1')dnl
 LOCAL_DOMAIN(`[127.0.0.1]')dnl
+RELAY_DOMAIN(`IPv6:0:0:0:0:0:0:0:1')dnl
+LOCAL_DOMAIN(`[IPv6:0:0:0:0:0:0:0:1]')dnl
